.github/workflows/unit-tests.yaml

```diff
@@ -5,21 +5,24 @@ on:
   - pull_request
```

Comment:
Since all `jobs` get run in parallel, I think we can move the jobs in here over to the currently-named `test-microk8s.yaml`.